### PR TITLE
Dockerize/26 - Dockerize the cra-template-truemark

### PR DESCRIPTION
**Pull request by anisha2lc**
## Trello Card Link:

- [dockerize the cra-template](https://trello.com/c/Txf4fi3H/26-dockerize-the-react-template-project)

## Tasks Done:

1.  dockerize the cra-template-truemark

## Screenshots:

server starting
image

unit test
image

e2e test

image

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+COPY package*.json ./
+
+# Copy app files
+COPY . .
+
+# Install project dependencies
+RUN npm install
+
+# Create a batch script to run both npm start,cypress and unit tests
+RUN echo "@echo off" > C:\start-and-test.bat \
```
Comment (coolprobn) [inline Dockerfile:18]: It should be possible to run the server, unit and e2e tests separately. You will have to research on this, using same docker for these all won't be fruitful as we want these commands to run off at a time. e.g. we want to run the server for doing the development work and use unit tests command only when we want to run unit tests after writing some tests

Comment (coolprobn) [reply Dockerfile:18]: You can take a new card for this purpose, let's create a new card for this and take in next sprint

Comment (anisha02lc) [reply Dockerfile:18]: @coolprobn dai, [Here](https://trello.com/c/MOq1Bz2x/33-run-the-server-unit-and-e2e-tests-separately-in-docker) is the card created for above task.

**Review by coolprobn — APPROVED** [review]
Great work, thanks!

**Pull request merged**